enos/enos-dev-scenario-pr-replication.hcl

```diff
@@ -139,7 +142,7 @@ scenario "dev_pr_replication" {
     // If you are using an ent edition, you will need a Vault license. Common convention
     // is to store it at ./support/vault.hclic, but you may change this path according
     // to your own preference.
-    vault_install_dir = matrix.artifact == "zip" ? var.vault_install_dir : global.vault_install_dir_packages[matrix.distro]
+    vault_install_dir = matrix.artifact == "zip" || matrix.artifact == "local" ? global.vault_install_dir["bundle"] : global.vault_install_dir["package"]
```

Comment:
Based on this, it seems like we have no real use of `vault_install_dir` being a variable at all since we ignore it and use our global map. Seems like we should remove it entirely?

Reply:
Our global map uses the artifact_type (bundle or package), which we don't have in this scenario, so this is like a re-mapping of our matrix.artifact to those.